### PR TITLE
missing update to the schema for "check all"

### PR DESCRIPTION
**Pull request by Geal**

### CODE REVIEW

File: schema.proto [hunk 1]
```diff
@@ -75,6 +75,12 @@ message RuleV2 {
 
 message CheckV2 {
   repeated RuleV2 queries = 1;
+  option Kind kind = 2;
```
Comment (divarvel) [inline schema.proto:78]:
```suggestion
  optional Kind kind = 2;
```

**Comment by divarvel** [comment]
apart from the typo, LGTM

File: schema.proto [hunk 2]
```diff
@@ -75,6 +75,12 @@ message RuleV2 {
...
+
+  enum Kind {
+    None = 0;
```
Comment (divarvel) [inline schema.proto:81]: wouldn't `any` be more suited? (not super important since it's only internal, but it's always better to strive for consistency)

Comment (Geal) [reply schema.proto:81]: right, that would make more sense

Comment (Geal) [reply schema.proto]: I think I messed up twice when copying, it was "One" not "None"

**Pull request merged**